CryptoTechReminderSystem.Main/Program.cs

```diff
@@ -42,7 +42,8 @@ public ReminderRegistry()
                 "https://api.harvestapp.com/",
                 Environment.GetEnvironmentVariable("HARVEST_TOKEN"),
                 Environment.GetEnvironmentVariable("HARVEST_ACCOUNT_ID"),
-                Environment.GetEnvironmentVariable("HARVEST_USER_AGENT")
+                Environment.GetEnvironmentVariable("HARVEST_USER_AGENT"),
+                Environment.GetEnvironmentVariable("HARVEST_ROLES")
```

Comment:
Maybe we could call these "HARVEST_DEVELOPER_ROLES" ?

Reply:
```suggestion
                Environment.GetEnvironmentVariable("HARVEST_DEVELOPER_ROLES")
```

Reply:
Done and done. 
In https://github.com/chubberlisk/crypto_tech/pull/31/commits/38522bc09cab41b8624ea7171b41a1fd591126b7